src/geode/geometry/distance.cpp

```diff
@@ -939,19 +948,19 @@ namespace geode
     std::tuple< double, Point3D > point_triangle_signed_distance(
         const Point3D& point, const Triangle3D& triangle )
     {
-        Point3D nearest_point;
-        double distance;
-        std::tie( distance, nearest_point ) =
-            point_triangle_distance( point, triangle );
-        const Vector3D proj2point{ nearest_point, point };
-        // Tetra facet normals point towards inside
-        if( const auto normal = triangle.normal() )
+        if( const auto pivot_and_normal = triangle.pivot_and_normal() )
         {
-            const auto signed_distance =
-                proj2point.dot( normal.value() ) <= 0 ? distance : -distance;
-            return std::make_tuple( signed_distance, nearest_point );
+            const auto output = pivot_point_triangle_distance(
+                point, triangle, pivot_and_normal->first );
+            const auto& nearest_point = std::get< 1 >( output );
+            const Vector3D proj2point{ nearest_point, point };
+            if( proj2point.dot( pivot_and_normal->second ) <= 0 )
+            {
+                return output;
+            }
+            return std::make_tuple( -std::get< 0 >( output ), nearest_point );
         }
-        return std::make_tuple( distance, nearest_point );
+        return no_pivot_point_triangle_distance( point, triangle );
```

Comment:
I do not understand: the - side is towards the normal side and the + side opposite? Strange definition or I've missed something?

Reply:
indeed but I did not change this part